src/components/HomeTab/index.jsx

```diff
@@ -23,7 +23,7 @@ const STATUS_COLOR_MAP = {
   success: 'rgba(40, 167, 69, 0.7)'
 }
 
-export class HomeTab extends React.Component {
+export class HomeTab extends React.PureComponent {

```

Comment:
Recommended reading: https://reactjs.org/docs/optimizing-performance.html#avoid-reconciliation

Reply:
That's correct. So if `props` have the same values as previous, this component will not even call `render`. `PureComponent` should be safe anytime the rendered DOM will be exactly the same given the same `props`. With a plain `Component`, `render` is always called whenever `props` or `state` change (which happens everytime a parent component called it's `render`), and then the new DOM is compared to the previous DOM to see if updates are actually needed.